javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,17 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (!class07Students.includes(studentName) && getNumberOfStudents()) {
+    class07Students.push(studentName);
+    console.log(`${studentName} added to class07`);
+  } else if (class07Students.includes(studentName)) {
+    console.log(`Student ${studentName} already in class07`);
+  }
+}
```

Comment:
Did you try the cases of adding an empty `studentName`? and the Queen of Denmark?